packages/block-library/src/shortcode/block.json

```
@@ -15,7 +15,10 @@
 	"supports": {
 		"className": false,
 		"customClassName": false,
-		"html": false
+		"html": false,
+		"interactivity": {
+			"clientNavigation": true
+		}
```

I'd keep this one as non-compatible, as we don't know if some shortcode will use JS.

```suggestion
		"html": false
```

---

Done